src/main/asciidoc/types/index.adoc

```diff
@@ -1,19 +1,50 @@
 [[types]]
 = Types
 
-The Neo4j types:
+[abstract]
+--
+Neo4j has a type system.
+Users want to send parameters pass parameters and process results.
```

Comment:
How about: `The Neo4j type system is used for parameters to and results from the database.`